src/main/asciidoc/query-by-example.adoc

```diff
@@ -186,3 +191,28 @@ The following table describes the scope of the various `ExampleMatcher` settings
 | Property path
 
 |===
+
+[[query-by-example.fluent]]
+== Fluent API
+
+`QueryByExampleExecutor` offers one more method, which we did not mention so far: `<S extends T, R> R findBy(Example<S> example, Function<FluentQuery.FetchableFluentQuery<S>, R> queryFunction)`.
+As with other methods it executes a query derived from an `Example`.
+But with the second argument you can control aspects of that execution, that you can't control dynamically otherwise.
```

Comment:
Change to:

However, with the second argument, you can control aspects of that execution that you cannot dynamically control otherwise.